product.py

```diff
@@ -0,0 +1,138 @@
+# This file is part of Tryton.  The COPYRIGHT file at the top level of
+# this repository contains the full copyright notices and license terms.
+from trytond.model import ModelSQL, ModelView, fields
+from trytond.pool import PoolMeta, Pool
+from trytond.pyson import Eval
+from trytond.exceptions import UserError
+from trytond.i18n import gettext
+
+
+class Certification(ModelSQL, ModelView):
+    "Certification"
+    __name__ = 'agronomics.certification'
+
+    number = fields.Char('Number')
+    date = fields.Date('Date')
+
+    def get_rec_name(self, name):
```

Comment:
Better set model's _rec_name because as implemented, searcher will not work.

For example: https://github.com/tryton/account_stock_landed_cost/blob/develop/account.py#L88